src/m365/spfx/commands/project/project-github-workflow-add.ts

```diff
@@ -150,6 +150,15 @@ class SpfxProjectGithubWorkflowAddCommand extends BaseProjectCommand {
       workflow.on.workflow_dispatch = null;
     }
 
+    const version = this.getProjectVersion();
+    if (!version) {
+      throw `Unable to determine the version of the current SharePoint Framework project`;
+    }
+
+    if (version.startsWith('1.18.')) {
```

Comment:
Should we not invert this? Use Node 18 by default and check for version 1.18 or lower to select node 16.

Reply:
>.. check for version 1.18 or lower to select node 16

hmm you mean rather we should check for SPFx 1.17.4 or lower right?

Reply:
done